pkg/cloudprovider/aws/fleet/packing/packing.go

```diff
@@ -16,17 +16,21 @@ package packing
 
 import (
 	"context"
+	"errors"
+	"fmt"
+	"sort"
 
 	"github.com/aws/aws-sdk-go/service/ec2/ec2iface"
 	"go.uber.org/zap"
 	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
 )
 
 type PodPacker struct {
```

Comment:
Does this type need to be public since we have the `Packer` interface?

Reply:
Yeah PodPacker can be private, thanks.